src/contentscript/injector.ts

```diff
@@ -561,10 +571,14 @@ export class Injector {
       }
 
       if (newBranch) {
-        addEvent(
-          'Branch resolving',
-          `Resolver of "${manifest.name}" defined the "${newBranch}" branch`
-        )
+        const notificationService = new NotificationService()
```

Comment:
Do not instance `const notificationService = new NotificationService()` in content script!
It's background's service. Call `createNotification` via `const { createNotification } = await initBGFunctions(browser)`

Reply:
+